portability-types-common/src/main/java/org/datatransferproject/types/common/models/media/MediaAlbum.java

```diff
@@ -1,16 +1,20 @@
 package org.datatransferproject.types.common.models.media;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.MoreObjects;
 import com.google.common.base.Preconditions;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 import java.util.stream.Collectors;
+import org.datatransferproject.types.common.ImportableItem;
 import org.datatransferproject.types.common.models.photos.PhotoAlbum;
 
-public class MediaAlbum {
+import javax.annotation.Nonnull;
+
+public class MediaAlbum implements ImportableItem {
```

Comment:
This model isn't really imported anywhere (in _this_ repo), but I don't see how making it an `ImportableItem` can hurt us.